docs/contribute/general/pr_workflow.rst

```diff
@@ -0,0 +1,128 @@
+
+.. role:: require-review
+   :class: sd-badge pst-badge github-red
+
+.. role:: drafting-phase
+   :class: sd-badge pst-badge github-purple
+
+.. role:: ready-for-pickup
+   :class: sd-badge pst-badge github-blue
+
+
+Development Workflow
+====================
+
+All pyfar related projects and packages are developed openly on GitHub.
+Contributions are managed through issues and pull requests, allowing communication
+between users, developers, and maintainers.
+Pull requests can be proposed from forks of the repository for which a contribution
+is to be committed.
+For contributors who are part of the pyfar organization it is recommended to work on feature
+branches on the repository directly.
+
+
+Workflow in a Nutshell
+----------------------
+
+The workflow can very briefly be summarized as:
+
+- Start a new feature branch for each set of edits that you do.
+- Hack away!
+- Push your feature branch to your own Github repo, and create a pull request.
+- Request a review and wait for feedback
+- If required, include the feedback from the review and update your code.
+- A maintainer will merge your PR if two reviewers approved.
+
+This way of working helps to keep work well organized.
+
+More Detailed Instructions
+--------------------------
+
+For more specific instructions for contributing to pyfar packages, please check out the following:
+
+- `Style and design guidelines <../packages/style_design.html>`_.
+- `Documentation guidelines <../packages/documentation.html>`_.
+- `Testing guidelines <../packages/testing.html>`_.
+
+The documentation of each package further provides instructions including the respective commands and tools, please refer to the guide of the package you want to contribute to.
+See for example the guidelines for `contributing to pyfar <https://pyfar.readthedocs.io/en/stable/contributing.html>`_
+
+Similarly, contribution guidelines exist for the *pyfar gallery* and the *open educational resources*:
+
+- `Contribution guidelines for the gallery <../contribution_gallery.html>`_.
+- `Contribution guidelines for the open educational resources <../contribution_oer.html>`_.
+
+
+Submitting a Pull Request
+-------------------------
+
+To get your changes merged into a pyfar repository or project, you need to submit a pull request (PR) on GitHub.
+A general guide on PRs can be found on the `GitHub documentation <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests?platform=linux>`_.
+
+Before Submitting a Pull Request
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please make sure to meet the following criteria:
+
+- Ensure that the PR targets the correct branch, i.e. ``main`` if you are proposing a bugfix or improvements to the documentation, or ``develop`` if you propose a new feature or enhancement of existing code.
+- Pyfar projects follow `semantic versioning <../packages/releasing.html#versioning>`_. This means that
```

Comment:
I would suggest to add a rule here to use a descriptive name for the branch, e.g. feature/..., bugfix/...